src/aznfswatchdog

```diff
@@ -137,7 +137,7 @@ conntrack -L > /dev/null
 
 # conntrack timewait timeout higher than the TCP timewait timeout value isn't very valuable.
 conntrack_timeo_timew=$(cat /proc/sys/net/netfilter/nf_conntrack_tcp_timeout_time_wait 2>/dev/null)
-if [ $? -eq 0 -a $conntrack_timeo_timew -gt $AZNFS_TIMEWAIT_TIMEOUT ]; then
```

Comment:
this is actually a correct statement. There are two reasons it might be failing:
1. Older versions of bash may by not handling it fine. I don't know about this, but this is a possibility.
2. The second possibility is that one of $conntrack_timeo_timew and $AZNFS_TIMEWAIT_TIMEOUT are null. Since AZNFS_TIMEWAIT_TIMEOUT is defined explicitly, this means conntrack_timeo_timew could be the one undefined, which means the above statement is yielding an empty value. This means netfilter module is not loaded and/or not present. In this case even the current code that you updated will also fail.

let's discuss this